app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -12,6 +12,7 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     require_nested :MiddlewareDatasource
     require_nested :MiddlewareMessaging
     require_nested :MiddlewareServer
+    require_nested :RefreshWorker
```

Comment:
You can always use `rebase -i HEAD~5` to re-arrange your commits (and merge that back into the commit it was removed from).
Since this commit is not something that you really want standalone as a commit.